uint32 handle
uint32 size
uint32 pid
---
int32 result

